### DISCUSSION TITLE
Behaviours are available for Windows build but not for mobile

### DISCUSSION THREAD

**Discussion by olivegamestudio** [unoplatform/uno#9142] [Q&A]
### Current behavior

Add the package Uno.Microsoft.Xaml.Behaviors.Interactivity.WinUI to the platform projects.

Add the following namespaces to the XAML:

    xmlns:Core="using:Microsoft.Xaml.Interactions.Core"
    xmlns:Interactivity="using:Microsoft.Xaml.Interactivity"
    xmlns:Media="using:Microsoft.Xaml.Interactions.Media"

Add the following to the XAML:

        <Interactivity:Interaction.Behaviors>
            <Interactions:EventTriggerBehavior EventName="TouchDown">
                <Media:ControlStoryboardAction Storyboard="{StaticResource Pressed}" ControlStoryboardOption="Play" />
            </Interactions:EventTriggerBehavior>
            <Interactions:EventTriggerBehavior EventName="TouchUp">
                <Media:ControlStoryboardAction Storyboard="{StaticResource Released}" ControlStoryboardOption="Play" />
            </Interactions:EventTriggerBehavior>
        </Interactivity:Interaction.Behaviors>

The windows project builds fine, but the Mobile ones do not find the interactions classes.

### Expected behavior

The behaviours are picked up like the Windows version.

**Comment by jeromelaban** [comment]
Thanks for the report! Could you provide a repro sample of the issue?
